jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/RunCommandDialog.kt

```diff
@@ -46,26 +52,27 @@ class RunCommandDialog(private val project: Project, private val container: Cont
 
     override fun createCenterPanel(): JComponent = panel {
         row(message("ecs.execute_command_task.label")) {
-            resourceSelector(
-                EcsResources.listTasks(container.service.clusterArn(), container.service.serviceArn()),
-                connectionSettings,
-                ::task
-            ).constraints(growX, pushX)
-                .growPolicy(GrowPolicy.MEDIUM_TEXT)
-                .withErrorOnApplyIf(message("ecs.execute_command_task_comboBox_empty")) { it.selected().isNullOrEmpty() }
+            cell(taskList)
+                .bindItem(::task)
+                .errorOnApply(message("ecs.execute_command_task_comboBox_empty")) {
+                    it.selected().isNullOrEmpty()
+                }.columns(30)
         }
         row(message("ecs.execute_command.label")) {
-            comboBox(commandList, ::command)
-                .constraints(grow)
-                .withErrorOnApplyIf(message("ecs.execute_command_no_command")) { it.item.isNullOrEmpty() }
+            comboBox(commandList).bindItem({ command }, {
+                if (it != null) {
+                    command = it
+                }
+            })
+                .errorOnApply(message("ecs.execute_command_no_command")) { it.item.isNullOrEmpty() }
                 .applyToComponent {
                     isEditable = true
                     selectedIndex = -1
                     editor = object : BasicComboBoxEditor.UIResource() {
                         override fun createEditorComponent() =
                             JBTextField().also { it.emptyText.text = message("ecs.execute_command_run_command_default_text") }
                     }
-                }
+                }.resizableColumn().align(Align.FILL)
```

Comment:
do we need all these resizable columns?

Reply:
Nope, I thought resizable column was a prereq for align, deleted them!